pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -18,37 +18,39 @@
 , libpng
 , python3
 , zlib
+, simde
 , bashInteractive
 , zsh
 , fish
 , nixosTests
-, go
+, go_1_22
```

Comment:
Why change from the default go? I don't see anything about this in the release notes.